Packs/ServiceNow/Scripts/ServiceNowTroubleshoot/SCRIPT_OUTPUT_FOR_DOC_REVIEW.md

```diff
@@ -0,0 +1,15 @@
+### Enabled Instances Health Information 
+
+
+|Instance Name|Last Pull Time|Names of Active Incidents Created 30 days ago|Number of Incidents Pulled in Last Fetch|Query|Size In Bytes|Total Active Incidents Created 30 days ago|
+|---|---|---|---|---|---|---|
+| ServiceNow_v2_instance_1 | 2025-01-29T11:03:45.122073608Z | ServiceNow Incident INC0041801,<br>ServiceNow Incident INC0041816| 5 | stateNOT IN6,7 | 3105 | 10 |
+ 
+ --- 
+ ### Disabled instances with Active Incidents Created 30 days ago
+|Instance|Names Active Incidents Created 30 days ago|Total|
+|---|---|---|
+| ServiceNow_v2_instance_1_copy_1 | ServiceNow Incident INC0033378<br>ServiceNow Incident INC0033356<br>ServiceNow Incident INC0033354  | 15 |
+
+### Note: The active incidents, created 30 days ago and listed in the tables for both enabled and disabled instances, are still being mirrored.
+ ### Please be aware that irrelevant active incidents may cause system overload. It is recommended to close them.
```

Comment:
What are irrelevant active incidents? It's not clear. Also, delete "Please be aware that"

Reply:
agreed over slack:
If the issue is no longer relevant and does not require further attention it is recommended to close related incidents promptly to prevent potential system overload